rules/windows/file/file_event/file_event_win_one_extension_files_in_susp_locations.yml

```diff
@@ -0,0 +1,26 @@
+title: OneNote Attachment File Dropped In Suspicious Location
+id: 7fd164ba-126a-4d9c-9392-0d4f7c243df0
+status: experimental
+description: Detects creation of files with the ".one" extension in suspicious or uncommon locations. This could be a sign of attackers abusing OneNote attachments
+references:
+    - https://www.bleepingcomputer.com/news/security/hackers-now-use-microsoft-onenote-attachments-to-spread-malware/
+    - https://blog.osarmor.com/319/onenote-attachment-delivers-asyncrat-malware/
+author: Nasreddine Bencherchali
+date: 2023/01/22
+tags:
+    - attack.defense_evasion
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection:
+        TargetFilename|contains:
+            - '\AppData\Local\Temp\'
+            - '\Users\Public\'
+            - '\Windows\Temp\'
+            - 'C:\Temp\'
```

Comment:
It is all the `\Temp\` folder we can looking at

Reply:
Can you please explain? Do you mean I should add more paths?